adapters/from_sql.py

```diff
@@ -48,7 +41,7 @@ def read_releases(release_data, airflow_db):
     release.labels = [item.dag_id]
     release.state = state
     if green_sha is None:
-      continue
+      release.links = [{'name': 'no links available', 'url': 'https://youtu.be/dQw4w9WgXcQ'}]
```

Comment:
Can we have url set to None, and have the UI not do a link in that case ?

Reply:
Done